org.hikounomizu.HikouNoMizu.yaml

```diff
@@ -0,0 +1,47 @@
+app-id: org.hikounomizu.HikouNoMizu
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: hikounomizu
+finish-args:
+    - '--device=all'
+    - '--share=ipc'
+    - '--socket=x11'
+    - '--socket=pulseaudio'
+    - '--persist=.hikounomizu'
+cleanup:
+    - /include
+    - '*.la'
+    - '*.a'
+modules:
+    - 'shared-modules/glu/glu-9.json'
+    - name: tinyxml
+      sources:
+          - type: archive
+            url: http://mirrors.kodi.tv/build-deps/sources/tinyxml-2.6.2_2.tar.gz
+            sha256: 8164c9ad48b9028667768a584d62f7760cfbfb90d0dd6214ad174403058da10c
+          - type: script
+            dest-filename: autogen.sh
+            commands:
+                - 'autoreconf -vfi'
+
+    - name: hikounomizu
+      buildsystem: cmake
+      post-install:
+          - 'install -Dm644 org.hikounomizu.HikouNoMizu.svg /app/share/icons/hicolor/scalable/apps/org.hikounomizu.HikouNoMizu.svg'
+          - 'install -Dm644 org.hikounomizu.HikouNoMizu.desktop /app/share/applications/org.hikounomizu.HikouNoMizu.desktop'
+          - 'install -Dm644 org.hikounomizu.HikouNoMizu.appdata.xml /app/share/metainfo/org.hikounomizu.HikouNoMizu.appdata.xml'
+
+      sources:
+          - type: archive
+            url: https://download.tuxfamily.org/hnm/0.8.91/hikounomizu-0.8.91-src-withdata.tar.bz2
+            sha256: a8a92f95dbb80a3d66b8dc8e54634d583a2e956f41dbd8566679dbb53cae115b
+
+          - type: file
+            path: org.hikounomizu.HikouNoMizu.svg
+
+          - type: file
+            path: org.hikounomizu.HikouNoMizu.desktop
+
+          - type: file
+            path: org.hikounomizu.HikouNoMizu.appdata.xml
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html